src/lib.rs

```diff
@@ -93,14 +92,12 @@ pub fn parse_module(
 /// Parse an individual module from an AST, returning the module.
 pub fn parse_module_from_ast(
   specifier: &ModuleSpecifier,
-  kind: ModuleKind,
   maybe_headers: Option<&HashMap<String, String>>,
   parsed_source: &deno_ast::ParsedSource,
   maybe_resolver: Option<&dyn Resolver>,
-) -> Module {
-  graph::parse_module_from_module_info(
+) -> EsmModule {
```

Comment:
Do we use this API to parse and analyze CommonJS files too? If so, seems strange to have it return `EsmModule`

Reply:
We do not at all and I don’t believe we should do that.

Reply:
I agree with that